webdriver/tests/element_click/element_click.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
```

Comment:
Please use a name for this helper which is identical to the actual command which is tested. This applies to all files.

Reply:
fixed to be element_click